test/Microsoft.AspNet.Mvc.FunctionalTests/BasicTests.cs

```diff
@@ -39,13 +40,11 @@ public class BasicTests
             var server = TestHelper.CreateServer(_app, SiteName, _configureServices);
             var client = server.CreateClient();
             var expectedMediaType = MediaTypeHeaderValue.Parse("text/html; charset=utf-8");
-
-            // The K runtime compiles every file under compiler/resources as a resource at runtime with the same name
-            // as the file name, in order to update a baseline you just need to change the file in that folder.
-            var expectedContent = await _resourcesAssembly.ReadResourceAsStringAsync("compiler/resources/BasicWebSite.Home.Index.html");
+            var outputFile = "compiler/resources/BasicWebSite.Home.Index.html";
```

Comment:
Our resources always come from `compiler/resources/*` would probably be easier to just have the `ResourceFile` utility read from `compiler/resources/` by default so you just have to provide the file and sub-folders.